### DISCUSSION TITLE
Visibility triggering

### DISCUSSION THREAD

**Discussion by kartun83** [zepp-health/zeppos-forum#180] [Developer Feedback]
In index.d.ts generated by zeus there is a method **IHmUIWidget.setVisibility**, but it doesn't work.
At [docs](https://docs.zepp.com/docs/reference/device-app-api/newAPI/ui/setProperty/#propertyid) it's said that visible property could be altered only alone, so it would be nice to have such alias.

**Comment by htoooth** [comment]
@kartun83
What kind of device do you have?

**Reply by kartun83** [reply]
gtr4
